prime-router/src/main/kotlin/cli/tests/SftpcheckTest.kt

```diff
@@ -0,0 +1,155 @@
+package gov.cdc.prime.router.cli.tests
+
+import com.github.kittinunf.fuel.Fuel
+import com.github.kittinunf.fuel.core.FuelError
+import com.github.kittinunf.fuel.core.Headers
+import com.github.kittinunf.fuel.core.Request
+import com.github.kittinunf.fuel.core.Response
+import com.github.kittinunf.fuel.core.extensions.authentication
+import com.github.kittinunf.fuel.json.responseJson
+import com.github.kittinunf.result.Result
+import gov.cdc.prime.router.cli.OktaCommand
+import gov.cdc.prime.router.cli.SettingCommand
+import gov.cdc.prime.router.common.Environment
+import org.apache.http.HttpStatus
+
+/**
```

Comment:
You need to move this kdoc to be right above the class.  Right now this kdoc will appear to be for the const.